packages/shared/src/components/RichTextEditor/plugins/ButtonPlugin/components/FloatingButton/useFloatingButtonEnter.ts

```diff
@@ -0,0 +1,25 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useEditorRef, useHotkeys } from '@udecode/plate';
+import { submitFloatingButton } from '../../transforms/submitFloatingButton';
+
+export const useFloatingButtonEnter = () => {
+    const editor = useEditorRef();
+
+    useHotkeys(
+        '*',
+        (e) => {
+            if (e.key !== 'Enter') {
```

Comment:
could it be directly: `if (e.key === 'Enter' && submitFloatingButton(editor)){e.preventDefault(); }`?